interop/xds/client/Dockerfile

```diff
@@ -24,13 +24,13 @@ COPY . .
 
 # Build a static binary without cgo so that we can copy just the binary in the
 # final image, and can get rid of Go compiler and gRPC-Go dependencies.
-RUN go build -tags osusergo,netgo interop/xds/client/client.go
+RUN cd interop/xds/client && go build -tags osusergo,netgo client.go
```

Comment:
How about `.`?

I think if you specify the file here and we add another file, it will break.

Reply:
Done.